docs/architecture/adr-023-data-fee-market.md

```diff
@@ -0,0 +1,60 @@
+# ADR 023: DA Fee Market
```

Comment:
[uber nit] can the ADR title and the filename match? Since DA != data, I got tripped up.

Either:
```suggestion
# ADR 023: Data Fee Market
```

or rename file to `adr-023-da-fee-market.md`